src/ts/secure-tunnel.ts

```diff
@@ -73,7 +73,7 @@ async function start(p: NodeJS.Process) {
   });
 
   if (!config.silent && config.infoUrl) {
-    opn(config.infoUrl.toString());
+    open(config.infoUrl.toString());
```

Comment:
`await` is required on call to async functions, otherwise there is no guarantee that the function exectues